source/agora/node/Registry.d

```diff
@@ -711,6 +705,13 @@ private struct TypedPayload
             assert(this.payload.addresses.length == 1);
             dg(ResourceRecord.make!(TYPE.CNAME)(name, this.payload.ttl,
                 Domain.fromString(this.payload.addresses[0].host)));
+
+            // Add URI record along with
+            dg(ResourceRecord.make!(TYPE.URI)(
+                Domain.fromString(format("_%s._tcp.%s", this.payload.addresses[0].schema, name.toString)),
```

Comment:
The first element is the name of the service, in our case, that should be Agora, shouldn't it ?